UnitsNet/GeneratedCode/Quantity.g.cs

```diff
@@ -451,7 +451,7 @@ public static bool TryFrom(double value, Enum? unit, [NotNullWhen(true)] out IQu
         /// <param name="quantityString">Quantity string representation, such as "1.5 kg". Must be compatible with given quantity type.</param>
         /// <param name="quantity">The resulting quantity if successful, otherwise <c>default</c>.</param>
         /// <returns>The parsed quantity.</returns>
-        public static bool TryParse(IFormatProvider? formatProvider, Type quantityType, string quantityString, [NotNullWhen(true)] out IQuantity? quantity)
+        public static bool TryParse(IFormatProvider? formatProvider, Type quantityType, string? quantityString, [NotNullWhen(true)] out IQuantity? quantity)
```

Comment:
I guess we could add `[NotNullWhen(true)]` to `string? quantityString` too.

Reply:
Seems you only edited the generated file by the way.
I'll make both changes in `StaticQuantityGenerator`.

Reply:
This appear to have caused a compilation error in the `AbbreviatedUnitsConverter` - I've pushed a fix on my branch (removing the extra `?`), but I don't think the PR can be reopened..